src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseService.java

```diff
@@ -95,6 +106,52 @@ public CaseDetails createCase(String authorization, String caseType, String even
         );
     }
 
+    public CaseData triggerEvent(String authorization, String caseId, String caseType,
+                                 CaseEvent eventName, String caseData) {
+        return triggerEvent(authorization, caseId, eventName, caseType, caseData);
+    }
+
+    public CaseData triggerEvent(String authorization, String caseId, CaseEvent eventName,
+                                 String caseType, String caseData) {
+        StartEventResponse startEventResponse = startUpdate(authorization, caseId, caseType, eventName);
+        return submitUpdate(authorization, caseId,
+                            caseDetailsConverter.caseDataContent(startEventResponse,
+                            employeeObjectMapper.getEmploymentCaseData(caseData)),
+                            caseType);
+    }
+
+    public StartEventResponse startUpdate(String authorization, String caseId, String caseType, CaseEvent eventName) {
+        String s2sToken = authTokenGenerator.generate();
+        UserDetails userDetails = idamClient.getUserDetails(authorization);
+
+        return ccdApiClient.startEventForCaseWorker(
+            authorization,
+            s2sToken,
+            userDetails.getId(),
+            JURISDICTION_ID,
+            caseType,
+            caseId,
+            eventName.name()
+        );
+    }
+
+    public CaseData submitUpdate(String authorization, String caseId,
+                                 CaseDataContent caseDataContent, String caseType) {
+        UserDetails userDetails = idamClient.getUserDetails(authorization);
+        String s2sToken = authTokenGenerator.generate();
+        CaseDetails caseDetails = ccdApiClient.submitEventForCaseWorker(
+            authorization,
+            s2sToken,
+            userDetails.getId(),
+            JURISDICTION_ID,
+            caseType,
+            caseId,
+            true,
+            caseDataContent
+        );
+        return caseDetailsConverter.toCaseData(caseDetails);
+    }
+
     private Et1CaseData getEmploymentCaseData(String caseData) {
```

Comment:
This can be removed now you've put it into a helper class

Reply:
I will remove this in future sprint as it was breaking one of the createCase junit test suite.